packages/commonwealth/server/router.ts

```diff
@@ -664,16 +667,31 @@ function setupRouter(
   );
 
   // rules
+  router.post(
+    '/getRules',
+    passport.authenticate('jwt', { session: false }),
+    getRules.bind(this, models)
+  );
   router.post(
     '/createRule',
     passport.authenticate('jwt', { session: false }),
     createRule.bind(this, models)
   );
+  router.post(
+    '/editRule',
+    passport.authenticate('jwt', { session: false }),
+    editRule.bind(this, models)
+  );
   router.post(
     '/deleteRule',
     passport.authenticate('jwt', { session: false }),
     deleteRule.bind(this, models)
   );
+  router.post(
+    '/checkAddressAgainstRule',
+    passport.authenticate('jwt', { session: false }),
```

Comment:
Equally, if we're going to make this route (whose name I dislike -- we should name it something more simple / straightforward) gated by user, we should just check the user's accounts, rather than arbitrary ones.